pyproject.toml

```diff
@@ -24,7 +24,7 @@ pygobject = "^3.34.0"
 # Optional (mainly plugins)
 musicbrainzngs = { version = "0.*", optional = true }
 dbus-python = { version = "*", optional = true }
-soco = { version = "^0.22.2", optional = true }
+soco = { version = "^0.28.0", optional = true }
```

Comment:
Do we _need_ this higher version? I definitely haven't used any of its features

Happy for it to _incidentally_ be higher (i.e. the lock file specifies it, but the interface (i.e. here) IMO should keep only what it requires

Reply:
No idea, I'm just updating everything. Should I revert this?

Reply:
reverted